### PR TITLE
Hw-4 Complete

### PR DESCRIPTION
**Pull request by SuperWinterMopper**
Spirit User search page complete.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,373 @@
...
-}
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mrs+Sheppards&display=swap');
+
+@media (min-width: 601px) {
+    * {
+        font-family: Montserrat;
+    }
+
+    :root {
```
Comment (jmsoltys) [inline src/App.css:9]: love to see that design system color scheme 🤧 so proud

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,373 @@
...
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .search-container .search-field h1 {
+        font-size: 36px;
+    }
+
+    .search-container .search-field .search-bar {
```
Comment (jmsoltys) [inline src/App.css:152]: love to see the use of subclasses here!

Comment (jmsoltys) [reply src/App.css:152]: however, if your class name, such as "search-bar" is unique, then I don't think you have to list all of the classes "above it" in the hierarchy just to be able to change the search-bar class. In other words, if no other class is called search-bar, it's enough to just so .search-bar{} to apply CSS to it (usually we use subclasses if there's specific parts of another class that we want to change)

for example, if we want to only change the h1 tags inside the search bar but not any other h1's, then subclasses are the way to go (which you already do, yippee!)

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,373 @@
...
+    }
+
+    footer h5 {
+        color: var(--Main-Yellow, #FFBD12);
+        font-feature-settings: 'liga' off, 'clig' off;
+        font-family: "Mrs Sheppards";
+        font-size: 25px;
+        font-weight: 400; 
+    }
+}
```
Comment (jmsoltys) [inline src/App.css:373]: your website looks CLEEEEEAN, amazing job once again!

**Comment by amyzliao** [comment]
You are free to merge this branch into main. We recommend you do so immediately, before you make any changes to your codebase for this assignment or future assignments.

If you have NOT started on assignment 5, the following steps will help you avoid tricky merge conflicts:
- If you have made local changes to your `hw-4` branch that are not published to github, first commit those changes and publish them to github. That should update this PR with your latest changes.
1. Merge this PR into main
2. Switch to the main branch: `git checkout main`
3. Download the latest main which should now have all your `hw-4` commits: `git pull`
4. Delete your local `hw-4` branch: `git branch -d hw-4`

If you HAVE started on assignment 5, the following steps will help you avoid tricky merge conflicts:
- **DO NOT MAKE ANY CHANGES TO YOUR `hw-4` BRANCH**. If you have already made changes to `hw-4`, you will have to resolve merge conflicts at step 6.
1. Merge this PR into main
2. Commit all your current changes for assignment 5 to the `hw-5` branch. Afterwards, make sure when you run `git status`, it says "nothing to commit, working tree clean".
3. Switch back to the main branch: `git checkout main`
4. Download the latest main which should now have all your `hw-4` commits: `git pull`
5. Switch back to your `hw-5` branch: `git checkout hw-5`
6. Rebase off of main: `git rebase main`
    - This is the trickiest bit. Ideally there should not be any conflicts.
    - If you have conflicts, the terminal output will tell you exactly which files conflict. Go to those files and resolve the conflicts. When you're done, run `git add . && git rebase --continue`
    - Either the rebase will finish or it will give you more merge conflicts to resolve. If it gives you more conflicts to resolve, repeat the above bullet point. It might make you resolve the same conflict more than once; that is normal behavior.
    - After you are fully finished with the rebase, run your code again and verify that it has the expected behavior. Bugs may have emerged during the rebase.
7. Force push your changes to the `hw-5` branch to github:
    - If this branch does not yet exist on github: `git push -u origin hw-5`
    - If this branch already exists on github: `git push --force-with-lease`

If you run into any issues, DM amy, ethan, or aanand on discord.

**Pull request merged**